pom.xml

```diff
@@ -318,70 +319,6 @@
         </configuration>
       </plugin>
 
-      <plugin>
-        <groupId>com.mycila</groupId>
-        <artifactId>license-maven-plugin</artifactId>
```

Comment:
note: this is moved to a profile disabled on skipTests, as it makes a lot of other things easier